Makefile

```diff
@@ -15,7 +15,12 @@ all: manager
 
 # Run tests
 test: generate fmt vet manifests
-	go test -v -covermode=atomic -coverprofile=coverage.out -race ./...
+	ginkgo -r --randomizeAllSpecs --randomizeSuites --failOnPending --cover -coverprofile=../coverage.out --trace --race --progress
```

Comment:
Does this give better output than the trash built in?

Reply:
Yes, it also gives you an idea about how many tests you are doing with a nice progress bar